native_client/dotnet/DeepSpeechConsole/Program.cs

```diff
@@ -95,7 +106,12 @@ static void Main(string[] args)
 
                         stopwatch.Start();
 
-                        string speechResult = sttClient.SpeechToText(waveBuffer.ShortBuffer, Convert.ToUInt32(waveBuffer.MaxSize / 2), 16000);
+                        string speechResult;
+                        if (extended != null) {
+                            speechResult = metadataToString(sttClient.SpeechToTextWithMetadata(waveBuffer.ShortBuffer, Convert.ToUInt32(waveBuffer.MaxSize / 2), 16000));
```

Comment:
So @carlfm01 here we are leaking since we don't free `Metadata`, though I don't know how to do it with the current `FreeMetadata(IntPtr ptr);` :)